index2.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Hobbies and Interests</title>
+  <link rel="stylesheet" type="text/css" href="css/style.css">
+</head>
+
+<body>
+
+<h2>
+  
+  Hobbies and Interests:
+</h2>
+
+
+<img class="2" src="Images/Churchill2.jpg">
```

Comment:
:hammer: The image isn't loading on the page. There is a case error here. Remember, `directory/image.jpg` is _totally_ different from `Directory/Image.jpg`.